span/src/dropless.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright Rust project developers under MIT or APACHE-2.0.
```

Comment:
Does this line pass our copyright check? I have a guess that we need to run it by someone.

Reply:
I'd be surprised if it didn't; both MIT and ASL2 are compatible with GPL3, including with CLASSPATH exceptions.

Reply:
I suggest to double-check with @damons w.r.t. Aleo's "policy" on this.